Tests/ChiRelaxationTest/ChiRelaxationTest.cpp

```diff
@@ -0,0 +1,173 @@
+/* GRTeclyn
+ * Copyright 2022 The GRTL collaboration.
+ * Please refer to LICENSE in GRTeclyn's root directory.
+ */
+// Doctest header
+#include "doctest.h"
+
+// Test header
+#include "ChiRelaxationTest.hpp"
+
+// Common test headers
+#include "InitialData.hpp"
+#include "doctestCLIArgs.hpp"
+
+// GRTeclyn headers
+#include "ChiRelaxation.hpp"
+#include "DefaultPotential.hpp"
+#include "ScalarField.hpp"
+
+// AMReX headers
+#include "AMReX.H"
+#include "AMReX_FArrayBox.H"
+#include "AMReX_MultiFab.H"
+
+#ifdef AMREX_USE_HDF5
+#include "H5Cpp.h"
+#include <AMReX_PlotFileUtilHDF5.H>
+using namespace H5;
```

Comment:
I think we can remove this since `AMReX_PlotFileUtilHDF5.H` already includes the
HDF5 C header. Since the HDF5 library can be built without C++ support, we
shouldn't unnecessarily include it.

In general, I don't like `using namespace xxx` statements as well since this
breaks namespace clash protection.
```suggestion
#include <AMReX_PlotFileUtilHDF5.H>
```